cfme/fixtures/service_fixtures.py

```diff
@@ -20,22 +20,38 @@
 from cfme.utils.log import logger
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="function")
 def dialog(request, appliance):
     return _dialog(request, appliance)
 
 
 @pytest.fixture(scope="module")
+def dialog_modscope(request, appliance):
+    return _dialog(request, appliance)
+
+
+@pytest.fixture(scope="function")
 def catalog(request, appliance):
     return _catalog(request, appliance)
 
 
 @pytest.fixture(scope="module")
+def catalog_modscope(request, appliance):
+    return _catalog(request, appliance)
+
+
+@pytest.fixture(scope="function")
 def catalog_item(appliance, provider, provisioning, dialog, catalog):
     catalog_item = create_catalog_item(appliance, provider, provisioning, dialog, catalog)
     return catalog_item
 
 
+@pytest.fixture(scope="module")
+def catalog_item_modscope(appliance, provider, provisioning, dialog, catalog):
```

Comment:
@izapolsk i think `provisioning` will be in `function` scope as per #8736 so we can make `provisioning_modscope`. else it will give us error.

Reply:
@digitronik, I'm not entirely sure it will give us an error. May I ask you to take care of it if it occurs ?

Reply:
sure